src/lib/y2storage/boot_requirements_strategies/legacy.rb

```diff
@@ -148,7 +148,15 @@ def errors_on_gpt
         errors = []
 
         if grub_part_needed_in_gpt? && missing_partition_for?(grub_volume)
-          errors << SetupError.new(missing_volume: grub_volume)
+          message = _(
+            "There is no partition of type BIOS Boot.\n" \
+            "Although other configurations would eventually make possible\n" \
```

Comment:
missing `it`

Reply:
Personally, I find the message is a bit too long.

Reply:
There is no partition of type BIOS Boot.

It is strongly advised to use a BIOS Boot partition.

A setup without such a partition is not supported and may cause unnecessary problems later.

Reply:
BTW, your mockup shows no effect of the line breaks. Maybe html markup is needed?

Reply:
I guess I will just drop the line breaks, they would look weird in a `<li>` anyways.